frontend/src/components/GoalForm/ReadOnly.js

```diff
@@ -1,89 +1,133 @@
-import React, { useRef } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import ContextMenu from '../ContextMenu';
-import Modal from '../Modal';
 import ReadOnlyObjective from './ReadOnlyObjective';
 import './ReadOnly.scss';
 
-export default function ReadOnly({
+export function ReadOnlyGoal({
```

Comment:
I would personally prefer this in it's own file if possible

Reply:
Anything is possible. :astronaut: :new_moon:

Reply:
![th-2700848131](https://user-images.githubusercontent.com/3288586/184213964-f963d672-40ff-40e6-ac28-1090d947ed02.jpg)

Reply:
I knew you would come around. :alien: